### ISSUE TITLE
Support for finding Training models for Object Detecton 

### ISSUE THREAD

**Issue by takeofuture** [keras-team/autokeras#1236]
I see another issue thread which has been already closed.

So objectDetection models eventually supported, (I am not talking about pretrained model, I assume it is not the job for autokeras as AutoML as it main objective is to automatically search and find candidates for the models based on given data).
I am assuming given training dat asimilar like COCO or so (class label + (x, y, h, w) ) * xxx, then it find the good model for object detection automatically. Any update so far? I am looking forward to using it!

**Comment by haifeng-jin** [comment]
duplicate of #925
